### PR TITLE
Removed unused chainspecs

### PR DESCRIPTION
**Pull request by akorchyn**
Integrated DEX into sydney.
Prepared new chain spec to relaunch sydney
Removed unused chainspecs

### CODE REVIEW

File: node/src/runtime/mainnet.rs [hunk 1]
```diff
@@ -100,11 +95,53 @@ pub fn testnet_genesis(
...
+		.unwrap()
+		.into(),
 	};
+	let validator_identity: ValidatorIdentity = ValidatorIdentity {
+		id: parse_account_id("qHWFTG53dT7WvVa4HeGgrAwYNPDs6WFvzgkwxtJbQzJyjQH1S"),
+		session_keys,
+	};
+
+	let multisig_owners: Vec<_> = [
+		"5H9a1Q4rqzEK1SU5gFZBFjdBUEvCSGxJb7z9pRoE3veut153", // Raymond
```
Comment (uchu) [inline node/src/runtime/mainnet.rs:126]: I'd suggest not to commit names @akorchyn 

Lets just post it somewhere in our channels instead.

File: node/src/runtime/mainnet.rs [hunk 2]
```diff
@@ -114,17 +151,21 @@ pub fn testnet_genesis(
...
-			// Configure endowed accounts with initial balance of 1 << 60.
-			balances: endowed_accounts
-				.iter()
-				.cloned()
-				.map(|(k, endowment)| (k, endowment as u128 * GGX))
-				.collect(),
+			balances: [
+				(
+					multisig.clone(),
+					TOTAL_SUPPLY - (1100 * GGX) - (500 * GGX * multisig_owners.len() as u128),
```
Comment (uchu) [inline node/src/runtime/mainnet.rs:162]: @akorchyn So do we keep 1000 tokens for min staking?

File: node/src/runtime/mainnet.rs [hunk 3]
```diff
@@ -201,27 +223,39 @@ pub fn testnet_genesis(
...
 		},
 		eth_2_client: Eth2ClientConfig {
 			networks: vec![
-				(
-					webb_proposals::TypedChainId::Evm(1),
-					NetworkConfig::new(&Network::Mainnet),
-				),
 				(
 					webb_proposals::TypedChainId::Evm(5),
 					NetworkConfig::new(&Network::Goerli),
```
Comment (uchu) [inline node/src/runtime/mainnet.rs:261]: @akorchyn Is Goerli still in use?

**Review by uchu — CHANGES_REQUESTED** [review]
@akorchyn Some comments

File: node/src/runtime/mainnet.rs (hunk 1 above)
Comment (akorchyn) [reply node/src/runtime/mainnet.rs:126]: The code would be reverted, it just to create json

File: node/src/runtime/mainnet.rs (hunk 2 above)
Comment (akorchyn) [reply node/src/runtime/mainnet.rs:162]: Yeah :( it requires to careful changes in code

File: node/src/runtime/mainnet.rs (hunk 3 above)
Comment (akorchyn) [reply node/src/runtime/mainnet.rs:261]: the Goerli network was decided to come to an end and will be deprecated in April 2024, which will completely cease the support for the network.

Comment (akorchyn) [reply node/src/runtime/mainnet.rs:261]: But it's still running though

**Review by uchu — APPROVED** [review]

**Pull request merged**